### PR TITLE
fix(ssh_tunnel): fix test condition for existing config files

### PR DESCRIPTION
**Pull request by empwilli**

### CODE REVIEW

File: src/cli.rs [hunk 1]
```diff
@@ -300,8 +300,8 @@ pub enum SshConfig {
         #[arg(short = 'u', long = "user", default_value = "omnect")]
         username: String,
         /// optional: path where the ssh key pair, the certificates, and the
-        /// temporary ssh configuration is stored. Defaults to system local data
-        /// directories (e.g. ${XDG_RUNTIME_DIR}/omnect-cli on Linux).
+        /// temporary ssh configuration is stored. Defaults to system local
+        /// runtime directory (e.g. ${XDG_RUNTIME_DIR}/omnect-cli on Linux).
```
Comment (JanZachmann) [inline src/cli.rs:304]: Imho the comment is still wrong:
```
jan@JZ-Mint-I631:~$ echo ${XDG_RUNTIME_DIR}
/run/user/1000
```
vs. 
`~/.local/share/omnect-cli/config`

Comment (empwilli) [reply src/cli.rs:304]: ack, that one slipped through. It is fixed now.

File: src/cli.rs [hunk 2]
```diff
@@ -310,7 +310,7 @@ pub enum SshConfig {
         #[arg(short = 'k', long = "key")]
         priv_key_path: Option<PathBuf>,
         /// optional: path where the ssh configuration is stored. Defaults to system
-        /// local data directories (e.g. ${XDG_RUNTIME_DIR}/omnect-cli/ssh_config on
+        /// local runtime directory (e.g. ${XDG_RUNTIME_DIR}/omnect-cli/ssh_config on
```
Comment (JoergZeidler) [inline src/cli.rs:313]: ssh_config should be changed to config

Comment (empwilli) [reply src/cli.rs:313]: fixed

**Review by JanZachmann — APPROVED** [review]

**Review by JoergZeidler — APPROVED** [review]

**Review by JanZachmann — APPROVED** [review]

**Pull request merged**